src/app/query/stacks/bns/bns.query.ts

```diff
@@ -18,10 +21,16 @@ const bnsQueryOptions = {
   refetchOnReconnect: false,
 } as const;
 
-function getBnsNameFetcherFactory(client: StacksClient, limiter: RateLimiter) {
+type BnsNameFetcher = (address: string) => Promise<BnsNamesOwnByAddressResponse>;
+
+function getBnsNameFetcherFactory(
+  client: StacksClient,
+  limiter: RateLimiter,
+  isTestnet: boolean
```

Comment:
Passing in a bool flag makes it hard to read when invoked (reader needs look at implementation to understand

Reply:
changed to obj in clone